src/actions/signUpAction.ts

```diff
@@ -0,0 +1,9 @@
+import { AnyAction } from 'redux'
+import { ActionSignup } from '../models/actionModels/signup'
+
+export function autoPopulateEmail(email: string): ActionSignup | AnyAction {
+  return {
+    type: 'ACTION_SIGNUP',
+    payload: email,
+  }
+}
```

Comment:
Have you tried this with backend service. When you do a create account the details should get entered in DB in Create Account service and verify if the details are present in H2 DB and see the email is auto populated. You need to remove the hardcoded portions if any